guides/common/modules/proc_exporting-a-repository-in-a-syncable-format.adoc

```diff
@@ -0,0 +1,59 @@
+[id="Exporting_a_Repository_in_a_Syncable_Format_{context}"]
+= Exporting a Repository in a Syncable Format
+
+You can export the content of a repository in the Library environment of an organization of an organization to a syncabe format that you can use to create your custom CDN and synchronize the content from the custom CDN over HTTP/HTTPS.
+
+ifdef::satellite[]
+You can then serve the generated content using a local webserver on the importing {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to:
+
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}.
+* Update your CDN configuration to *Custom CDN*.
+* Set the CDN URL to point to the web server.
+* Optionally set a SSL CA Credential if the web server requires it.
+* Enable the Repository.
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
+endif::[]
+
+You can export the following content in a syncable format from {ProjectServer}:
+
+* RPM repositories
+* Kickstart repositories
+* File repositories
+
+You cannot export Ansible and Docker content.
+
+ifndef::satellite[]
+You cannot export DEB repositories from {ProjectServer}.
+endif::[]
+
+The export contains the following data:
+
+Directories containing the RPMs, *listing* files and metadata of the repository in yum format that can be used to synchronize in the importing {ProjectServer}.
+
+.Prerequisite
+* Ensure that you set download policy to *Immediate* for the repository within the Library lifecycle environment you export.
+For more information, see xref:Download_Policies_Overview_{context}[].
+
+.Procedure
+. Export a repository using the repository name or ID:
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer content-export complete repository \
+ --organization="_My_Organization_" \
```

Comment:
```suggestion
--organization="_My_Organization_" \
```

We mostly avoid whitespace before multi line commands. Please adjust all affected lines.